bin/coveralls

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env php
+<?php
+
+if (file_exists(__DIR__ . '/../vendor/autoload.php')) {
+    require __DIR__ . '/../vendor/autoload.php';
+} elseif (file_exists(__DIR__ . '/../../autoload.php')) {
+    require __DIR__ . '/../../autoload.php';
+} else {
+    die(
+        'You need to set up the project dependencies using the following commands:' . PHP_EOL .
+        'curl -s http://getcomposer.org/installer | php' . PHP_EOL .
+        'php composer.phar install' . PHP_EOL
+    );
+}
+
+use Satooshi\Bundle\CoverallsBundle\Console\Application;
+
+$rootDir = getcwd();
```

Comment:
Alright, this is simpler. Also I'm considering to provide a command line option to specify the rootDir in another PR.